tests/unit/controllers/SessionControllerTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace prime\tests\unit\controllers;
+
+use prime\controllers\SessionController;
+use prime\helpers\JobQueueProxy;
+use Yii;
+use yii\base\InvalidConfigException;
+use yii\filters\AccessControl;
+use yii\filters\AccessRule;
+use yii\web\ForbiddenHttpException;
+use yii\web\Request;
+use yii\web\User;
+
+/**
+ * @covers \prime\controllers\SessionController
+ */
+final class SessionControllerTest extends ControllerTest
+{
+    public function testConstructor(): void
+    {
+        $controller = $this->getController();
+        $this->assertInstanceOf(SessionController::class, $controller);
+    }
```

Comment:
What is this even testing? The `SessionController` class does not have a constructor.